packages/react-ui/__tests__/widgets/DrawingToolWidgetUI.test.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import { render, queryByAttribute, fireEvent } from '@testing-library/react';
+import DrawingToolWidgetUI from '../../src/widgets/DrawingToolWidgetUI';
+import { capitalize, SvgIcon } from '@material-ui/core';
+
+const CURSOR_ICON_ID = 'cursor-icon';
+
+const CursorIcon = (
```

Comment:
Let's include the icons in a separate module and export it from react-ui, so they can be used in widgets

Reply:
As you're going to export it we need to specify it belongs to the drawing tool.  Something like:

```
import {drawingToolIcons} from '@carto/react-ui'
drawingToolIcons.Polygon
```

Reply:
Done.